ansible/playbooks/AdoptOpenJDK_Unix_Playbook/main.yml

```diff
@@ -37,7 +37,8 @@
     - role: NVidia_Cuda_Toolkit   # OpenJ9
       tags: [build_tools, build_tools_openj9]
     - Superuser                   # AdoptOpenJDK Infrastructure
-    - Swap_File
+    - role: Swap_File
```

Comment:
What's the reason for not setting up a swap file on RISCV64?

Reply:
the swapfile steps failed on riscv, it seemed to be to do with the fact that we were using an sd card but not 100% sure??

Reply:
Shouldn't make any difference - what was the error?

Reply:
Can you confirm what the error was here @gdams - I've set up swapfiles in a similar way on many devices using SD cards. We should enhance the role (to be honest, that's not a bad idea anyway for other reasons) or justify why it's being excluded